src/components/attendee/Banner/Banner.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useCallback, useEffect } from "react";
+import { useHistory } from "react-router-dom";
+import { Button } from "components/attendee/Button";
+
+import { Banner as TBanner } from "types/banner";
+
+import { openUrl } from "utils/url";
+
+import { useShowHide } from "hooks/useShowHide";
+
+import { RenderMarkdown } from "components/organisms/RenderMarkdown";
+
+import CN from "./Banner.module.scss";
+
+export type BannerProps = {
+  banner: TBanner;
+  turnOnBlur: () => void;
+  turnOffBlur: () => void;
+};
+
+export const Banner: React.FC<BannerProps> = ({
+  banner,
+  turnOnBlur,
+  turnOffBlur,
+}) => {
+  const { isShown: isBannerShown, hide: closeBanner } = useShowHide(true);
+
+  console.log("Banner", banner);
+
+  const isBannerFullScreen = banner.isFullScreen;
+  const isWithButton = banner?.buttonDisplayText && banner?.isActionButton;
+  const isBannerCloaseable = !banner?.isForceFunnel;
+
+  useEffect(() => {
+    if (!isBannerFullScreen) return;
+
+    if (isBannerShown) {
+      turnOnBlur();
+    }
+  }, [isBannerFullScreen, turnOnBlur, isBannerShown]);
+
+  const { push: openUrlUsingRouter } = useHistory();
+
+  const hideBanner = useCallback(() => {
+    turnOffBlur();
+    closeBanner();
+  }, [turnOffBlur, closeBanner]);
+
+  const goToButtonDestination = useCallback(() => {
```

Comment:
`navigateToBannerDestination` ?
`gotoBannerDestination`?

Reply:
Fixed in https://github.com/sparkletown/sparkle/pull/2952/commits/38d6ac6a94dd7aa2549bf7fcb0ad9d2fc7a05a13